<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverStateTest</class>
 <widget class="QWidget" name="RecoverStateTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_trigger">
     <property name="text">
      <string>触发</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_recover_main">
     <property name="text">
      <string>恢复主状态</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_recover_child">
     <property name="text">
      <string>恢复子状态</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>229</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
